pages/common/fastmod.md

```diff
@@ -0,0 +1,21 @@
+# fastmod
+
+> A fast partial replacement for the codemod tool, replace and replace all in the whole codebase.
+> Regexes are match with the regex crate: <https://crates.io/crates/regex>.
```

Comment:
```suggestion
> Regexes are matched by Rust regex crate.
```

Reply:
Should I remove the link for the crate? I can add a link for its docs: https://docs.rs/regex/latest/regex/ ?

Reply:
The convention is using only one link per page

Reply:
Sorry, I added the link for the regex crate docs, I think it is better for quick access and to help the `fastmod`'s to get more info about the regex he is writing.

We can assume that 1 is nearly = 2 in this case... 😂